Engine/EditorUI/Panel/InspectorSubpanel/PanelMetaFile.cpp

```diff
@@ -34,13 +34,12 @@ void PanelMetaFile::Render(Metafile * metafile)
 	ImGui::Text(metafile->imported_file_path.c_str());
 
 	ImGui::Text("TimeStamp:"); ImGui::SameLine();
```

Comment:
@yalania Is this text `ImGui::Text("TimeStamp:");` still needed?

Reply:
I agree here the field is empty